.expeditor/end_to_end.pipeline.yml

```diff
@@ -380,19 +390,26 @@ steps:
           privileged: true
           environment:
             - BUILD_PKG_TARGET=x86_64-linux
-            - ACCEPTANCE_HAB_AUTH_TOKEN
 
+
+  # TODO (CM) : I think this isn't going to work properly
   - label: "[:linux: test_pkg_bulkupload]"
     command:
       - .expeditor/scripts/end_to_end/setup_environment.sh dev
       - test/end-to-end/test_pkg_bulkupload.sh
     expeditor:
+      secrets:
+        # This is an *Acceptance* Builder token; don't want this test
+        # actually uploading things to Production Builder, for safety.
+        PIPELINE_HAB_AUTH_TOKEN:
+          path: account/static/habitat/chef-ci
+          field: scotthain-sig-key
       executor:
         docker:
           privileged: true
           environment:
-            - ACCEPTANCE_HAB_AUTH_TOKEN
-            - HAB_BLDR_URL=https://bldr.acceptance.habitat.sh
+            - PIPELINE_HAB_AUTH_TOKEN
+            - PIPELINE_HAB_BLDR_URL=http://bldr.acceptance.habitat.sh
```

Comment:
http or https?

Reply:
Ugh... good catch on the `https`. Will fix!

And yes, I'm going to make sure the proper token is wired in for Acceptance before this is merged.